test/driver/exception_handling/run.t

```diff
@@ -124,3 +124,21 @@ and the whole AST is prepended with an error extension node.
   $ ./whole_file_exception.exe -embed-errors impl.ml
   Fatal error: exception Failure("An exception in a whole file transform")
   [2]
+
+
+4. Reporting Multiple Exceptions
+
+Just one exception is raised when -embed-errors is not added.
+  $ ./whole_file_multiple_errors.exe impl.ml
+  File "impl.ml", line 1, characters 0-12:
+  Error: SHOULD APPEAR FIRST
+  [1]
+
+When the argument `-embed-errors` is added, the exception is added to the existing errors and prepended to the ast.
+  $ ./whole_file_multiple_errors.exe -embed-errors impl.ml
+  [%%ocaml.error "SHOULD APPEAR FIRST"]
+  [%%ocaml.error "A second located error in a whole file transform"]
+  [%%ocaml.error "SHOULD APPEAR LAST"]
+  type a = int
+  type b = int[@@deriving deriver_raised_exception]
```

Comment:
Maybe a new `impl.ml` should be created, without this line.

Reply:
The file `impl.ml` is defined at lines [113 and 114 of this file](https://github.com/ocaml-ppx/ppxlib/pull/447/files/b04e47ed9d2e7706785f88fe13bf7c84ea13f74c#diff-264139ab557e4aad6831cc4158977efce83cb19d7079a2bce2bfa5d51c5e3f49R113-R114). Such a file is created once per example, overwriting the previous one. For instance an `impl.ml` file was already created at line [84-86](https://github.com/ocaml-ppx/ppxlib/pull/447/files/b04e47ed9d2e7706785f88fe13bf7c84ea13f74c#diff-264139ab557e4aad6831cc4158977efce83cb19d7079a2bce2bfa5d51c5e3f49R84-R86), overwritten at line 113-114.

It would be ideal to overwrite `impl.ml` one more time to have the file tailored to the example (e.g., without a mention to `deriver_raised_exception`)